nixops/backends/__init__.py

```diff
@@ -334,7 +334,8 @@ def run_command(self, command, **kwargs):
         # mainly operating in a chroot environment.
         if self.state == self.RESCUE:
             command = "export LANG= LC_ALL= LC_TIME=; " + command
-        return self.ssh.run_command(command, self.get_ssh_flags(), **kwargs)
+
+        return self.ssh.run_command(command, **kwargs)
```

Comment:
Why is `self.get_ssh_flags()` removed here?

Reply:
See the commit message for 574ba39. Although this is quick-and-dirty WIP code, I always try to write comprehensible commit messages, so `git blame` can help people understand changes.

> `MachineState.run_command()` passes SSH flags to `self.ssh.run_command()`.
However, `self.get_ssh_flags()` is already registered as a `ssh_flag_fun` in the
class `__init__()` function, so `ssh_util.SSH` already uses it to get the flags
when initiating a connection. This lead to the SSH flags being duplicated, which
causes an error for some flags (e.g. the `-J` flag, which can only be specified once).

Reply:
My bad -- I didn't pay enough attention. Great commit message!